projects/ion.md

```diff
@@ -16,7 +16,7 @@ ION is a blockchain based decentralized cryptocurrency that rewards network part
 ## Contacts
 [GitHub](https://github.com/ionomy)  
 [Bitcointalk](https://bitcointalk.org/index.php?topic=1443633.0)   
-[Facebook](https://www.facebook.com/ionomy/)
+[Facebook](https://www.facebook.com/ionomy/)  
```

Comment:
Two spaces in the end of line move the text to the new line. Before this small fix, links to Facebook and official forum where on the same line, now they are on two separate ones.